src/v/pandaproxy/test/consumer_group.cc

```diff
@@ -88,14 +94,11 @@ FIXTURE_TEST(pandaproxy_consumer_group, pandaproxy_test_fixture) {
           res.body.data(), ppj::create_consumer_response_handler());
         BOOST_REQUIRE(res_data.instance_id != kafka::no_member);
         member_id = res_data.instance_id;
-        BOOST_REQUIRE_EQUAL(
-          res_data.base_uri,
-          fmt::format(
-            "http://{}:{}/consumers/{}/instances/{}",
-            advertised_address.host(),
-            advertised_address.port(),
-            group_id(),
-            member_id()));
+        std::cout << res_data.base_uri << std::endl;
+        // TODO: replace this check when we will add context based rendering of
+        // consumer group member base_uri. Our http client doesn't set the
+        // `Host:...` HTTP header.
+        BOOST_REQUIRE_EQUAL(res_data.base_uri, "");
```

Comment:
Doesn't this break the ducktape tests?

Reply:
it indeed does, I've updated this PR with a version that fully supports advertised API.